.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Do we need this file?

Reply:
This file is not needed, husky adds `/.husky/_/,gitignore` to handle the ignoring.

Reply:
@danisharora099 not sure you saw.

Reply:
Yes, husky adds both files to handle the ignoring. 

This was generated by husky, and the comment was resolved after a conversation with @weboko.

Reply:
If we do remove this, an empty will be committed.